projects/dialogue_bias/util.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from collections import defaultdict
+from typing import Dict, List, Tuple
+
+import pandas as pd
+
+
+RACES_ETHNICITIES = ['hispanic', 'white', 'black', 'api', 'aian', '2prace']
```

Comment:
Are the spellings of the last two items here intentional?

Reply:
Yeah, that's how Tzioumis spelled them - adding a comment for this